maui-toolkit/TabView/Nested-Tabs.md

```diff
@@ -206,4 +206,22 @@ The following image illustrates the nested tab structure:
 
 ![Nested Tabs in .NET MAUI Tab View](images/Nested_Tab.png)
 
-N> View [sample](https://github.com/SyncfusionExamples/maui-toolkit-samples/tree/master/TabView/NestedTabViewSample) in GitHub.
\ No newline at end of file
+N> View [sample](https://github.com/SyncfusionExamples/maui-toolkit-samples/tree/master/TabView/NestedTabViewSample) in GitHub.
+
+## Enable virtualization
+
+EnableVirtualization enhances the initial loading performance of SfTabView. To enable this feature, set the `EnableVirtualization` property to `true.`
```

Comment:
Enabling virtualization improves the initial loading performance of SfTabView. To activate this feature, set the EnableVirtualization property to true.

Reply:
Updated the content.